.github/workflows/integration.yml

```
@@ -87,46 +87,46 @@ jobs:
       - name: "Check out the repo"
         uses: "actions/checkout@v4"
-      
+
       - name: "Setup node"
         uses: actions/setup-node@v2
         with:
           node-version: 20
-      
+
       - name: Install Rust 1.81.0
         uses: actions-rs/toolchain@v1
```

actionlint (1.7.4)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)